web/src/pages/Status/__tests__/StatusPage.test.jsx

```diff
@@ -108,12 +102,13 @@ const testTagsData = {
         scheduled: 4,
         completed: 1,
       },
+      service_ids: ["50604348-fd06-4152-afd1-2f3e73c4eb9f"],
     },
     {
-      tag_id: "56cfb764-e0ae-4acd-ad14-72312a30e17a",
-      tag_name: "setuptools:pypi:",
-      parent_id: "56cfb764-e0ae-4acd-ad14-72312a30e17a",
-      parent_name: "setuptools:pypi:",
+      package_id: "56cfb764-e0ae-4acd-ad14-72312a30e17a",
+      package_name: "setuptools:pypi:",
```

Comment:
```suggestion
      package_name: "setuptools",
```
package_nameに「:」を含むのは相応しくないです
94行目も同様